vendor/gopt/gopt-usage.c

```diff
@@ -1,83 +0,0 @@
-/* gopt-usage.c   PUBILC DOMAIN 2015   t.gopt@purposeful.co.uk */
```

Comment:
Why did you delete this file?

Reply:
It's unused, and gets caught by the `vendor/gopt/*.c`.
I could make the glob more specific if you prefer